tests/data/snake_oil_data.py

```diff
@@ -145,9 +163,14 @@ def content(self):
             "id": "SNAKE_OIL_GPR_DIFF",
             "has_observations": False,
         },
-        "FOPR": {
-            "name": "FOPR",
-            "id": "FOPR",
+        f"{INDEX_RESPONSE}": {
```

Comment:
I think : `f"{INDEX_RESPONSE}": {...}` is equivalent with `INDEX_RESPONSE: {...}`

Reply:
yes, of course, thank you : )
fixed!